site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -757,11 +765,14 @@ public function showStatus($gradeable_id) {
                         }
                         $sections['all']['total_components'] += $value * $num_components;
                         $sections['all']['graded_components'] += isset($graded_components[$key]) ? $graded_components[$key] : 0;
+                        $sections['all']['verified_components'] += isset($verified_components[$key]) ? $verified_components[$key] : 0;
```

Comment:
This is a great opportunity to use PHP's [null coalescing operator](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce)!

```suggestion
                        $sections['all']['verified_components'] += $verified_components[$key]) ?? 0;
```